src/test/java/be/syntra/devshop/DevshopFront/controllers/SearchControllerTest.java

```diff
@@ -46,203 +42,180 @@
     @MockBean
     private ProductService productService;
 
-    @MockBean
-    private ProductListCacheService productListCacheService;
-
     @MockBean
     private CartService cartService;
 
     @MockBean
     private SearchService searchService;
 
+    @MockBean
+    private ProductMapperUtil productMapperUtil;
+
     @Test
-    public void canDisplaySearchedForProductsTest() throws Exception {
+    void showSearchBarResultTest() throws Exception {
         // given
-        final String searchRequest = "product";
-        final String description = "another";
-        final List<Product> dummyProducts = getDummyNonArchivedProductList();
-        final ProductList dummyProductList = new ProductList(dummyProducts);
-        final CartDto dummyCart = getCartWithOneDummyProduct();
-        final SearchModel dummySearchModel = new SearchModel();
-        dummySearchModel.setSearchRequest(searchRequest);
-        dummySearchModel.setDescription(description);
-
-        when(productListCacheService.findBySearchRequest(any())).thenReturn(dummyProductList);
-        when(productListCacheService.filterByPrice(dummyProducts, dummySearchModel)).thenReturn(dummyProductList);
-        when(productListCacheService.searchForProductDescription(dummyProducts, dummySearchModel)).thenReturn(dummyProductList);
+        final String TEST_REQUEST = "testRequest";
```

Comment:
not a static String. Convention-wise, it shouldn't be allCaps

Reply:
idd, changed